tests/test_C3D8R_error_expected.py

```diff
@@ -0,0 +1,15 @@
+parameters = {
+	"results": [
```

Comment:
It would be nice to verify that the error occurs. I can't think of an easy way to do this with Abaverify in it's current form. I'll add an issue to Abaverify to note this is a needed feature.

Reply:
Agreed. If the number of frames could be accessed, that could work to confirm the special output frame was created. The test is designed to fail between frames 4/10 and 5/10, so if there are 5 frames, it failed and worked as intended. If there are 10 frames, no problem error occurred, and the test should fail.